.travis.yml

```diff
@@ -1,13 +1,23 @@
+# This Travis job script has been generated by a script via
+#
+#   haskell-ci 'travis' 'cabal.project'
+#
+# For more information, see https://github.com/haskell-CI/haskell-ci
+#
 language: c
-sudo: false
+dist: xenial
 
 git:
   submodules: false  # whether to recursively clone submodules
```

Comment:
You might need this as `true` to get riak `tools`?

Reply:
@tmcgilchrist Worth a shot but it was `false` before :)

@llelf's suggestion of `sudo sh` did not work, trying again with `sudo: false` this time (as before), I'm thinking it is going to to break again, though

Reply:
This was the problem after all. I guess mafia was taking care of cloning submodules previously.

Reply:
yeah `mafia` does that for you.